app/database/operator/server_data_operator/transformers/post.ts

```diff
@@ -167,7 +167,7 @@ export const transformSchedulePostsRecord = ({action, database, value}: Transfor
         scheduledPost.rootId = raw?.root_id ?? '';
         scheduledPost.message = raw?.message ?? '';
         scheduledPost.channelId = raw?.channel_id ?? '';
-        scheduledPost.files = raw?.files ?? emptyFileInfo;
+        scheduledPost.files = raw?.metadata?.files ?? emptyFileInfo;
```

Comment:
So based on this function I'm asuming we will never keep the values of existing records, is that true?